mindjunkies/accounts/tests.py

```
@@ -0,0 +1,56 @@
+        self.assertEqual(response.status_code, 200)
+        self.assertIn("course_list", response.context)  # Ensure template receives course_list
+        self.assertNotIn("enrolled_classes", response.context)
+        self.assertNotIn("teacher_classes", response.context)
+
+    def test_home_view_authenticated_no_courses(self):
+        """Test home view for an authenticated user with no enrollments"""
+        self.client.login(username="testuser", password="testpass")  # Log in user
+
+        response = self.client.get(reverse('home'))
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.context["enrolled_classes"], [])
```

Ruff (0.8.2)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)